kiali-server/values.yaml

```diff
@@ -87,7 +87,6 @@ deployment:
   tolerations: []
   version_label: ${HELM_IMAGE_TAG} # v1.39 # v1.39.0 # see: https://quay.io/repository/kiali/kiali?tab=tags
   view_only_mode: false
-
```

Comment:
don't delete this line - notice in this values yaml we have a blank line to separate the major groupings of settings so it is easy to visually see.